build/docker/Dockerfile

```
@@ -1,5 +1,7 @@
 FROM alpine:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)